vizro-core/docs/pages/user_guides/tabs.md

```diff
@@ -0,0 +1,121 @@
+# How to use tabs
+
+This guide shows you how to use tabs to organize your containers into subsections inside the dashboard.
+
+To add [`Tabs`][vizro.models.Tabs] to your page, do the following:
+
+1. Insert the [`Tabs`][vizro.models.Tabs] into the `components` argument of the [`Page`][vizro.models.Page]
+2. Insert your [`Containers`][vizro.models.Container] into the `components` argument of the [`Tabs`][vizro.models.Tabs]
+
+
```

Comment:
```suggestion

Note that by using the [`Tabs`][vizro.models.Tabs] model, the following holds true:
- [`Filters`][vizro.models.Filter] affect their targets independent of tabs, i.e. they filter even when the target is not visible
- the `title` of the [`Containers`][vizro.models.Container] inserted will become the title of the tab
- if you would like to add a title inside the tab, you can add a nested [`Container`][vizro.models.Container]
- it is possible to have multiple containers per [`Page`][vizro.models.Page]

```

Reply:
I like it! I've rephrased a bit and added some additional information:


```
By using the [`Tabs`][vizro.models.Tabs], the following applies:
- If targets are not specified, [`Filters`][vizro.models.Filter] affect all components on all tabs, i.e., they also filter the components on the non-opened tab
- The `title` of the outer [`Container`][vizro.models.Container] inserted into `Tabs.components` will be displayed as a tab label and the title will removed from the `Container`
- If you want to keep the `Container.title` inside the tab, you can apply custom CSS
- If you want to have a different `Container.title` inside the tab, you can add a nested [`Container`][vizro.models.Container]

```

Reply:
Sounds perfect! The only thing I would maybe change a little is the first bullet. The point about filters is true, but explains rather our filtering behaviour. I think the more interesting bit is that hidden tabs are filtered.

I assume this happens not only if no targets are specified (what we say), but also when a "currently hidden" target is specified (what we somewhat don't say). So maybe we can emphasize that a little more 🤔 Wdyt?

Reply:
I've rephrased a bit